sites/all/modules/hr/hr_datasets/js/hr_datasets.js

```diff
@@ -27,8 +27,18 @@ Drupal.behaviors.hrDatasets = {
       var models = response.result.results ? response.result.results : {};
       this.count = response.result.count;
       var fields = [];
-      _.each(models, function(resource){
-        fields.push({title: resource.title, url: 'https://data.hdx.rwlabs.org/dataset/' + resource.id});
+      _.each(models, function(resource){   
+        var date = new Date(resource.metadata_modified);
+        var month = ["JAN", "FEB", "MAR", "APR", "MAY", "JUNE", "JULY", "AUG", "SEPT", "OCT", "NOV", "DEC"];
+        var day = date.getDate();
+        var monthIndex = Drupal.t(date.getMonth());
```

Comment:
You are right, I missed on that, even though the dates displayed just fine. Pushing a commit to fix it. Thanks.